libraries/dateformatter/impl/src/main/kotlin/io/element/android/libraries/dateformatter/impl/DateFormatters.kt

```diff
@@ -20,7 +19,10 @@ import java.time.Period
 import java.util.Locale
 import javax.inject.Inject
 import kotlin.math.absoluteValue
+import kotlin.time.Clock
+import kotlin.time.ExperimentalTime
 
+@OptIn(ExperimentalTime::class)
```

Comment:
Could we OptIn for the whole project instead of adding manually everywhere?

Reply:
Good idea: 182909dd4a1de81eb44f443a11f6ab7c44fed320